curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfb6a35eacea3f48c6300b4.md

```diff
@@ -67,7 +67,7 @@ Your `figcaption` element's text should be `Cats love lasagna.` You have either
 
 ```js
 assert(
-  document.querySelector('figcaption').innerText.match(/Cats love lasagna.?$/i)
+  document.querySelector('figcaption').innerText.trim().match(/Cats love lasagna.?$/i)
```

Comment:
```suggestion
  document.querySelector('figcaption').innerText.trim().match(/^Cats love lasagna\.?$/i)
```
escaping the dot is a more needed change here, how that stayed like that for so long is a mistery